WORKSPACE

```diff
@@ -119,8 +128,11 @@ format_repositories()
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "45409e6c4f748baa9e05f8f6ab6efaa05739aa064e3ab94e5a1a09849c51806a",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.18.7/rules_go-0.18.7.tar.gz",
```

Comment:
Why did you upgrade rules_go? Because of skylib?

Reply:
yes